cf-execd/cf-execd-runner.c

```diff
@@ -583,6 +583,7 @@ static void MailResult(const ExecConfig *config, const char *file)
         struct stat statbuf;
         if (stat(file, &statbuf) == -1)
         {
+            Log(LOG_LEVEL_DEBUG, "Mail report: failed to stat file '%s' [errno: %d]", file, errno);
```

Comment:
I think this should be `LOG_LEVEL_ERR`. There's a similar place below logging that the file failed to open using the `ERR` level. And it makes sense to me that this should be reported not only when running in the `--debug` mode.